docs/guides/collecting-openshift-logs-with-victoria-logs/README.md

```diff
@@ -0,0 +1,183 @@
+* [Helm installed](https://helm.sh/docs/intro/install)
+
+## Install Red Hat OpenShift Logging operator 
+
+[Cluster logging operator](https://github.com/openshift/cluster-logging-operator) is a logging solution to support aggregated cluster logging. It is using [Vector](https://vector.dev/) for logs collection and shipping to remote storage.
+
+In order to install the operator, navigate to the OpenShift web console and select the `Operators` tab. Then, click on `OperatorHub` and search for `Red Hat OpenShift Logging`. Click on the `Red Hat OpenShift Logging` operator and then click on `Install`.
+
+![Install logging operator](install-oc-logging-operator.webp)
+
+
+## RBAC configuration
+
+Create a service account and cluster role binding for the service account to access the logs. 
+OpenShift provides separate `ClusterRoles` for monitoring of different types of logs: `audit`, `infrastructure` and `application`. 
+
+The following configuration will allow the service account to collect all types of logs:
+```yaml
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: victorialogs
+  namespace: openshift-logging
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vl-collect-infra
+subjects:
+  - kind: ServiceAccount
+    name: victorialogs
+    namespace: openshift-logging
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: collect-infrastructure-logs
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vl-collect-audit
+subjects:
+  - kind: ServiceAccount
+    name: victorialogs
+    namespace: openshift-logging
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: collect-audit-logs
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vl-collect-application
+subjects:
+  - kind: ServiceAccount
+    name: victorialogs
+    namespace: openshift-logging
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: collect-application-logs
+```
+
+Alternatively, you can use OpenShift console to create the service account and cluster role binding. 
+Navigate to `ServiceAccounts` and click on `Create Service Account`. Fill in the name `victorialogs` and namespace `openshift-logging` and click on `Create`. 
+Then, navigate to `RoleBindings` and create a binding for each `ClusterRole` for subject `victorialogs` in `openshift-logging` namespace.
+
+
+## Install Victoria Logs
+
+Add Victoria Metrics Helm [repository](https://github.com/VictoriaMetrics/helm-charts):
+```bash
+helm repo add vm https://victoriametrics.github.io/helm-charts/
+```
+
+Minimal configuration for VictoriaLogs running in OpenShift:
+```yaml
+securityContext:
+  enabled: true
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - ALL
+  readOnlyRootFilesystem: true
+
+podSecurityContext:
+  enabled: true
+  fsGroup: 1000660000
```

Comment:
These ID's look odd, why this particular number?

Reply:
AFAIR, on a test cluster with OpenShift local it was required to use a value higher than `1000000000`, so this was just a random number confirming the requirements.
Do you think it is better to set it to something like `1000000001` to make it less confusing?

Reply:
I don't think you need them at all? The files would be owned by a random ID, but they will have group set to `root` always, so as long as files are group readable/writable its gonna be fine. This way you'll be able to run with `restricted` SCC and avoid requiring `anyuid` SCC.

Otherwise you're attempting to fit into the restricted range, which is 1000000000+ default but may also be tuned by experienced admins, so in some cases this won't work. In order to make sure it _always_ sets the correct UID you should add [anyuid rolebinding](https://examples.openshift.pub/deploy/scc-anyuid/#past-438) and set UID any pretty value like 1000

Reply:
Opened a PR [here](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/8344) to remove these IDs and also add a note that configuration might vary based on an actual cluster configuration. Thanks for the details!